worker-document/src/main/java/com/hpe/caf/worker/document/config/DocumentWorkerConfiguration.java

```diff
@@ -58,6 +58,11 @@
      */
     private ScriptCachingConfiguration scriptCaching;
 
+    /**
+     * Enable returning Exception on failure
+     */
+    private boolean enableExceptionOnFailure;
```

Comment:
I'm not sure about the name of this.  The .js file in the archetype hasn't been updated yet.  Does this name fit in with the rest of the configuration?

Reply:
If I have to add this config in archetype then I will have to make it configurable and default it to false right ? Subsequently make the deploy change for bulk-indexer to turn it true

Reply:
> If I have to add this config in archetype then I will have to make it configurable and default it to false right ?

Right

> Subsequently make the deploy change for bulk-indexer to turn it true

Not necessarily - it is OK for a worker to use a custom configuration file.  The artifact files are only defaults.